keyboards/soy20/keymaps/via/keymap.c

```diff
@@ -0,0 +1,33 @@
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	[0] = LAYOUT(
+		KC_PMNS, KC_PAST, KC_PSLS, MO(1),
+		KC_PPLS, KC_P9, KC_P8, KC_P7,
+		KC_TAB, KC_P5, KC_P6, KC_P4,
+		KC_PENT, KC_P3, KC_P2, KC_P1,
+		KC_DEL, KC_SCLN, KC_PDOT, KC_P0),
+
+	[1] = LAYOUT(
+		KC_NLCK, KC_PSCR, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, RESET),
+
+	[2] = LAYOUT(
```

Comment:
```suggestion
	[2] = LAYOUT_ortho_5x4(
```

Reply:
Does this affect functionality or is it a standards adherence?

Reply:
No change in functionality, but by doing this and adding `LAYOUTS = blah` to rules.mk, users with other boards with this layout will be able to use their existing keymaps via the [Community Layouts](https://docs.qmk.fm/#/feature_layouts?id=layouts-using-a-keymap-with-multiple-keyboards) feature.

Reply:
Cool, thanks for the heads up.